app/models/comment.rb

```diff
@@ -1,4 +1,5 @@
 class Comment < ApplicationRecord
-  belongs_to :author, class_name: 'User'
   belongs_to :post
+  belongs_to :author, class_name: 'User'
```

Comment:
- [ ] Let's address this issue of duplicate code in comments model. This line of code should only appear once `belongs_to :author, class_name: 'User'`